test/crowdsale/CapStagedCrowdsale.test.js

```diff
@@ -0,0 +1,63 @@
+const CapStagedCrowdsale = artifacts.require('CapStagedCrowdsale');
+const BigNumber = web3.BigNumber;
+const SimpleToken = artifacts.require('SimpleToken');
+require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('CapStagedCrowdsale', function ([_, owner, wallet, investor]) {
+  const _rate = new BigNumber(1000);
+  const _value = web3.toWei('1', 'ether');//  10ether
+  const stage1Limit = web3.toWei('1', 'ether');//  10ether
+  const stage1Rate = new BigNumber(40);
+  const stage2Limit = web3.toWei('2', 'ether');//  10ether //  20ether
+  const stage2Rate = new BigNumber(30);
+  const stage3Limit = web3.toWei('3', 'ether');//  10ether//  30ether
+  const stage3Rate = new BigNumber(20);
+  const stLimits = [stage1Limit, stage2Limit, stage3Limit];
+  const stRates = [stage1Rate, stage2Rate, stage3Rate];
+  const tokenSupply = new BigNumber(160000000000000000000);
+  beforeEach(async function () {
+    this._token = await SimpleToken.new({ from: owner });
+    this._csale = await CapStagedCrowdsale.new(_rate, wallet, this._token.address, stLimits, stRates, { from: owner });
+    await this._token.transfer(this._csale.address, tokenSupply, { from: owner });
+  });
+  describe('should accept payments in stage 1 (rate 40)', function () {
```

Comment:
instead of duplicating your "describe" and "it" blocks, try doing something like

```
context('in stage 1', function () {
  it('should accept purchases at rate 40', async function () {
  })
})
```

which reads much more clearly

Reply:
I've removed describes and change them to context